openkfw/Oscar
# Get map layers with or without geodata

## Description


Refactor getMapLayers function to return map layers with or without geoData. 

## How has this been tested?




1. Create initial-data-load/data/testCountry4 folder
2. Copy  initial-data-load/data/Sample/MapLayers.yml to testCountry4 folder
3. Remove geoReferenceId from layers config
4. Add COUNTRY=testCountry4 to .env file
5. Run ./start.sh
6. Run ./runinitialload.sh
7. In UI map layers click on Select layers
8. You can see layers there, click on some layer
8. This will cause TypeError. This should never happen because this functionality is not supposed for mapLayers we have there now. These mapLayers should have geoReferenceId always.
9. Check the same for Sample, to see if layers with geoData are uploaded properly

## Screenshots (if appropriate):

## Types of changes




## Checklist:
